.github/workflows/ci.yml

```diff
@@ -206,10 +206,14 @@ jobs:
         working-directory: build
         run: |
           ${{ steps.python-path.outputs.python-path }} -m pip install gcovr
-          ${{ steps.python-path.outputs.python-path }} -m gcovr . -r ../src \
+          ${{ steps.python-path.outputs.python-path }} -m gcovr -r .. \
```

Comment:
If it was the id related issue, I think this change should be reverted. Otherwise the coverage paths need to be excluded, which feels like a backwards solution.

Reply:
What coverage paths need to be excluded? This is the command we use in Sunshine (not saying it's 100% right, but it does work).

Reply:
> What coverage paths need to be excluded? This is the command we use in Sunshine (not saying it's 100% right, but it does work).

These paths:
```
--exclude '.*_deps/.*' \
--exclude '.*tests/.*' \
--exclude '.*third-party/.*' \
```

The previous approach was "run coverage here and map `../src` files only", with your changes it's more like "run coverage here and map `..` files only, but exclude <insert there exclude dirs>", which to be seems kinda strange.

Anyway, this is not a hill I'm willing to die on - you make the final decision and let's just roll with it.

Reply:
Okay, that makes sense. let me try this logic in Sunshine.